### ISSUE TITLE
Providing detections on the test set using train model

### ISSUE THREAD

**Issue by TimoRST** [NVlabs/FocalFormer3D#42]
Hi,

I've been working with the result files you provided. You've shared the detection results for a model trained on train and a model trained on trainval on the training and validation set. However, the readme only contains the result file for the trainval model on the test set.

Would it be possible to also share the result file for the test set from the model that was trained solely on train? This would be very helpful for my work.

Thanks!

**Comment by chenyilun95** [comment]
You might check out this question https://github.com/NVlabs/FocalFormer3D/issues/9

**Comment by TimoRST** [comment]
Hi, thanks for the quick response!
That's where we found the detections on the train and val set, but as far as I can see, there are no detections on the test set from the train only model available.

**Comment by chenyilun95** [comment]
No we did not run it. You should inference the pre-trained model on the train set yourself.

**Comment by TimoRST** [comment]
Thank you for the clarification. It seems that setting up the environment is quite difficult due to the outdated versions. I was hoping that either your team or a user of your model might have already generated these detections or could produce them easily.

**Comment by chenyilun95** [comment]
Thanks for clarification. Unfortunately, we don’t have the bandwidth to run these detections ourselves at the moment.

**Comment by weijielong** [comment]
> Thank you for the clarification. It seems that setting up the environment is quite difficult due to the outdated versions. I was hoping that either your team or a user of your model might have already generated these detections or could produce them easily.

Hi,timorst!

Have you completed the work on the result file for the test set?If possible, would you mind sharing it?

BEST，
jielong
